src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -1603,6 +1603,35 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_activation_eltwise_diff_sizes, :
     conv_eltw_test_params{ CASE_CONV_ELTW_FP32_8, 3, 3, 4 },
 }));
 
+class conv_fp32_group_conv_eltwise_sum : public ConvEltwTest {};
+TEST_P(conv_fp32_group_conv_eltwise_sum, basic) {
+    auto p = GetParam();
+
+    implementation_desc conv_impl = { format::bfyx, "convolution_gpu_bfyx_os_iyx_osv16", impl_types::ocl };
+    bo_fused.set_option(build_option::force_implementations({ { "conv_prim", conv_impl } }));
+
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        data("weights", get_mem(get_weights_layout(p))),
+        data("eltwise_data", get_mem(layout{ p.data_type, p.input_format, p.eltw_shape })),
+        convolution("conv_prim", input_info("input"), { "weights" }, {}, p.groups, p.stride, p.pad, p.dilation, p.out_shape, p.data_type, true),
+        eltwise("sum", { input_info("conv_prim"), input_info("eltwise_data") }, eltwise_mode::sum, data_types::f32),
+        reorder("reorder_bfyx", input_info("sum"), p.default_format, data_types::f32)
+    );
+
+    tolerance = default_tolerance(p.default_type);
+    execute(p);
+}
+
+// in_shape; out_shape; eltw_shape; kernel; stride; pad; dilation; groups; data_type; input_format; weights_type; weights_format; default_type; default_format;
+#define CASE_GROUP_CONV_ELTW_FP32_1 { 1, 48, 3, 3 }, { 1, 48, 3, 3 }, { 1, 48, 1, 1 }, { 1, 1, 3, 3 }, { 1, 1 }, { 1, 1 }, { 1, 1 }, 16, data_types::f32, format::bfyx, data_types::f32, format::g_os_iyx_osv16, data_types::f32, format::bfyx
+#define CASE_GROUP_CONV_ELTW_FP32_2 { 1, 48, 3, 3 }, { 1, 48, 3, 3 }, { 1, 48, 3, 3 }, { 1, 1, 3, 3 }, { 1, 1 }, { 1, 1 }, { 1, 1 }, 16, data_types::f32, format::bfyx, data_types::f32, format::g_os_iyx_osv16, data_types::f32, format::bfyx
+
+INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_group_conv_eltwise_sum, ::testing::ValuesIn(std::vector<conv_eltw_test_params>{
+    conv_eltw_test_params{ CASE_GROUP_CONV_ELTW_FP32_1, 2, 2, 3 },
+    conv_eltw_test_params{ CASE_GROUP_CONV_ELTW_FP32_2, 3, 2, 3 },
```

Comment:
This test passes even without your fix. Please confirm that it fails without your patch.

Reply:
Updated. This test will be failed with --device_suffix=1 without my patch.